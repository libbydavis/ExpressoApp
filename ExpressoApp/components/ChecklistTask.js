import React from 'react';
import {View, StyleSheet, Text} from 'react-native';

const CheckListTask = (props) => {
  return (
    <View style={styles.checklistTask}>
      <Text style={styles.checklistText}>{props.text}</Text>
      <Text>X</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  checklistTask: {
    backgroundColor: '#c9c9c9',
    padding: 10,
    margin: 5,
    borderRadius: 10,
    flexDirection: 'row',
  },
  checklistText: {
    color: '#383838',
    marginRight: 170,
  },
});

export default CheckListTask;
