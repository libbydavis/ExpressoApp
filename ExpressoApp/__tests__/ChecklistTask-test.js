// __tests__/ChecklistTask-test.js
import React from 'react';
import renderer from 'react-test-renderer';
import ChecklistTask from '../components/ChecklistTask';


test('renders correctly', () => {
    const checklistTask = renderer.create(<ChecklistTask text={"apple"}/>).toJSON();
    expect(checklistTask).toMatchSnapshot();
});
