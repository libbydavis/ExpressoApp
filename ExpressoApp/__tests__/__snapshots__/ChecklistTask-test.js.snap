// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#c9c9c9",
      "borderRadius": 10,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "margin": 5,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#383838",
      }
    }
  >
    apple
  </Text>
  <Text>
    X
  </Text>
</View>
`;
